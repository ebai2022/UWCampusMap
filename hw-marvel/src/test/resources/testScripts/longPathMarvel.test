# Testing a long path in the full marvel data set

LoadGraph g marvel.csv

FindPath g SACRIFICER SUMMER-KLEIN