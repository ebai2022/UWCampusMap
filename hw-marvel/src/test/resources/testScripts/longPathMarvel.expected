# Testing a long path in the full marvel data set

loaded graph g

path from SACRIFICER to SUMMER-KLEIN:
SACRIFICER to HAWK via AA2-28
HAWK to BARON-STRUCKER/WOLFG via ST-156
BARON-STRUCKER/WOLFG to HANNIBAL/HENRY-KING via NS-2
HANNIBAL/HENRY-KING to SUMMER-KLEIN via JIM-520